### ISSUE TITLE
Request: Scrollable Frame

### ISSUE THREAD

**Issue by f1refa11** [TomSchimansky/CustomTkinter#627]

**Comment by Th3o-D** [comment]
You can find [here](https://gist.github.com/JackTheEngineer/81df334f3dcff09fd19e4169dd560c59) a Tkinter scrollable frame that works just fine (Linux and Windows only). As mentioned in the code, there are a few lines to comment/uncomment depending on the OS on which you are using this code, or you can also add your own OS checker to the code, here is an example :

```python
import platform
if platform.system() == "Windows":
   # code for windows
else:
   # code for linux
```

As it is made with Tkinter, it will also work with CustomTkinter.

**Comment by TomSchimansky** [comment]
CTkScrollableFrame is now implemented with version 5.1.0.
